src/jdk.jartool/share/classes/sun/tools/jar/resources/jar.properties

```diff
@@ -290,6 +294,10 @@ main.help.opt.create.update.index=\
 \ Operation modifiers valid only in create, update, and generate-index mode:\n
 main.help.opt.create.update.index.no-compress=\
 \  -0, --no-compress          Store only; use no ZIP compression
+main.help.opt.create.update.index.date=\
+\      --date=TIMESTAMP       The timestamp in ISO-8601 extended offset date-time with\n\
+\                             optional time-zone format, to use for the timestamps of\n\
+\                             entries, eg. "2022-02-12T12:30:00-05:00"
```

Comment:
The updated wording looks much better. Just a minor nit, it should be "e.g." rather than "eg.".

Reply:
ah yes, should have known that, and I did Latin at school !

Reply:
fixed